keyboards/HP69/config.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 Ananya Kirti
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x416B
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0011
+#define MANUFACTURER    Desiboards
+#define PRODUCT         HP69
+#define DESCRIPTION     A 69 layout keyboard with apple bottom row
```

Comment:
```suggestion
```

Reply:
removed in lastest commit along with name changed to lowercase